compat.js

```diff
@@ -0,0 +1,11 @@
+export function macCompat() {
+  return (code, window, dir) => {
+    if (
+      window.navigator.platform.indexOf('Mac') === 0 &&
+      code.indexOf('meta+ctrl') === -1
+    ) {
+      return code.replace(...(dir ? ['ctrl', 'meta'] : ['meta', 'ctrl']))
```

Comment:
Do you have reason to have `dir` arg?

The code looks too complex for me

Reply:
It may not be a good reason, but I want to use the same transformer code in `hotkey.js` and in `getHotKeyHint`. However, they should work in different "directions":

- In `hotkey.js` it should transform meta+b into ctrl+b
- In `getHotKeyHint` it should transform ctrl+b into meta+b

Reply:
Right, we need to update `getHotKeyHint()` too (in this PR, otherwise the functions are not compatible)

Reply:
It's updated, take a look here: https://github.com/ai/keyux/pull/15/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R28-R30